src/assets/scss/_base.scss

```diff
@@ -5,4 +5,4 @@ body {
         overflow: hidden !important;
         height: 100% !important;
     }
-}
+}
```

Comment:
Needs end of line character, this shouldn't be a changed file after you do that.